app/views/distributions/_distribution.html.erb

```diff
@@ -0,0 +1,135 @@
+<header class="download-home-header jumbotron jumbotron-fluid bg-<%= @colour %> text-center pb-0">
+  <div class="container">
+    <div class="page-heading pb-5">
+      <% if @distro_type == "leap" %>
+        <%= leap_image_tag alt: "Leap Logo", class: "animated jackInTheBox", width: "256", height: "256" %>
+      <% elsif @distro_type == "kubic" %>
+        <%= image_tag 'distributions/kubic-white.svg', alt: "Kubic Logo", class: "animated zoomInDown", width: "256", height: "256" %>
+      <% else %>
+        <%= image_tag 'distributions/tumbleweed-white.svg', alt: "Tumbleweed Logo", class: "animated zoomInDown", width: "256", height: "256" %>
+      <% end %>
+      <h1 class="display-3 text-white">
+        <% if @distro_type == "leap" %>
+          openSUSE Leap <%= @version %>
+        <% elsif @distro_type == "kubic" %>
+          openSUSE Kubic
+        <% else %>
+          openSUSE Tumbleweed
+        <% end %>
+      </h1>
+    </div>
+  </div>
+  <div class="border-bottom" role="tablist">
+    <ul class="container nav nav-tabs nav-justified border-0">
+      <% @yaml_data&.each do |tab| %>
+      
+      <li class="nav-item">
+        <a class="nav-link h-100<% if @yaml_data[0]["name"] == tab["name"] %> active<% end %>" data-toggle="tab" href="#<%= tab["name"] %>-ports" role="tab">
+          <%= _(tab["name"]) %>
+          <% tab["arches"]&.each do |arch| %>
+            <span class="badge badge-<%= arch["color"] %>"><%= arch["name"] %></span>
+          <% end %>
+        </a>
+      </li>
+      <% end %>
+    </ul>
+  </div>
+</header>
+
+<section class="mb-5">
+  <div class="container">
+    <div class="tab-content">
+      <% @yaml_data&.each do |tab| %>
+      <div class="tab-pane <% if @yaml_data[0]["name"] == tab["name"] %>active<% end %>" id="<%= tab["name"] %>-ports" role="tabpanel">
+        <% if tab["info"] %>
+          <div class="alert alert-<%= tab["info-colour"] %>">
+            <%= markdownify( _(tab["info"]) ) %>
+          </div>
+        <% end %>
+        <% tab["arches"]&.each do |arch| %>
+        <h3 class="my-3 text-center"><%= arch["name"] %></h3>
+        <div class="row">
+          <% arch["types"]&.each do |type| %>
```

Comment:
[nitpick] do not use `&.` , use `try` instead (it is more readable) :see_no_evil: